WalletWasabi/BuyAnything/BuyAnythingManager.cs

```diff
@@ -122,7 +122,7 @@ private async Task CheckUpdateInOrderStatusAsync(ConversationUpdateTrack track,
 
 				await SendSystemChatLinesAsync(track,
 					ConvertOfferDetailToMessages(order),
-					new OfferCarrier(order.LineItems.Select(x => new OfferItem(x.Quantity, x.Label, x.UnitPrice, x.TotalPrice)), order.ShippingCosts),
+					new OfferCarrier(order.LineItems.Select(x => new OfferItem(x.Quantity, x.Label, x.UnitPrice, x.TotalPrice)), GetShippingCostFromOrder(order)),
```

Comment:
Arguably could call `GetShippingCostFromOrder(order)` here, save it in a variable and send it to `ConvertOfferDetailToMessages` as second parameter and use it in `OfferCarrier` as well, if it makes sense for others.